From 438bcfe48d8545373700e140cad9aca930efc23a Mon Sep 17 00:00:00 2001
From: RickyWu-wiwynn <ricky_cx_wu@wiwynn.com>
Date: Wed, 17 Jul 2024 11:33:26 +0800
Subject: [PATCH 4/7] configurations: Revise the Terminus NIC sensor name

1. Revise the Terminus NIC temperature sensor name to NICX_TEMP_C.
2. Revise the temperature threshold.

Change-Id: I46a611d4a4c2a29f320baedcdbe1c9112898de4a
---
 configurations/meta/terminus_100g_nic_tsff.json | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/configurations/meta/terminus_100g_nic_tsff.json b/configurations/meta/terminus_100g_nic_tsff.json
index e4f92d7..261ccd6 100644
--- a/configurations/meta/terminus_100g_nic_tsff.json
+++ b/configurations/meta/terminus_100g_nic_tsff.json
@@ -3,7 +3,7 @@
         {
             "Address": "$address",
             "Bus": "$bus",
-            "Name": "Terminus NIC FRU $bus",
+            "Name": "Terminus NIC FRU $bus % 24",
             "Type": "EEPROM"
         },
         {
@@ -12,7 +12,7 @@
             "Labels": [
                 "temp1"
             ],
-            "Name": "NIC TEMP $bus",
+            "Name": "NIC$bus % 24 TEMP_C",
             "Thresholds": [
                 {
                     "Direction": "greater than",
@@ -32,7 +32,7 @@
             "Type": "EMC1403"
         }
     ],
-    "Name": "Terminus NIC FRU $bus",
+    "Name": "Terminus NIC FRU $bus % 24",
     "Probe": "xyz.openbmc_project.FruDevice({'BOARD_PRODUCT_NAME': 'Terminus .*100G NIC TSFF'})",
     "Type": "Board",
     "xyz.openbmc_project.Inventory.Decorator.Asset": {
-- 
2.53.0-Meta

